bracex/__main__.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env/python
+"""
+Expands a bash-style brace expression, and outputs each expansion.
+
+Licensed under MIT
+Copyright (c) 2018 - 2020 Isaac Muse <isaacmuse@gmail.com>
+Copyright (c) 2021 Alex Willmer <alex@moreati.org.uk>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software,
+and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions
+of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+IN THE SOFTWARE.
+"""
+
+import argparse
+
+import bracex
+
+
+def main(argv=None):
+    """Accept command line arguments and output brace expansion to stdout."""
+    parser = argparse.ArgumentParser(
+        prog='python -mbracex',
+        description=__doc__.lstrip('\n').split('\n')[0],
```

Comment:
While clever, I'd prefer we just copy the string here. It's a lot of work to get a single short string 🙂. It'll also be easier for people to understand in general.